### PR TITLE
Added tests for object service dashboard page

### PR DESCRIPTION
**Pull request by bipuladh**
Integration for Object Service Dashboard
Todo:
 - [x] Add "id" for better for CSS selection.

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+  });
+
+  it('Checks whether MCG is healthy', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    expect(clusterHealth.isPresent()).toBeFalsy();
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Why are we expecting it to be false?

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  });
+
+  it('Checks for correct Efficiency Ratio', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Why wait for the visibility of obcCount to validate Efficiency Ratio?

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import {
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const BROWSER_TIMEOUT = 30000;
+
+describe('Test Object Service Dashboard Page', () => {
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Its better to move ```await browser.get(`${appHost}/dashboards/object-service`);``` and the visibility check to `beforeEach` block to remove duplication.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  it('Create an OBC and test equality', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    // Go to create form page
+    await browser.get(
```
[the remark this answers is not in the document]
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Yes but will be following a different approach, will use `kubectl` to create obc.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+
+    await browser.get(`${appHost}/dashboards/object-service`);
+    await browser.wait(until.visibilityOf(obcCount));
+    await browser.sleep(5000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+  });
+
+  it('Checks whether MCG is healthy', async () => {
```
[the remark this answers is not in the document]
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: The order of the test should not matter. If unhealthy even creation of OBC will fail.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts (hunk 1 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: because there is no message for healthy status only for unhealthy and error states.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts (hunk 2 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: ack

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+  obcCount,
+  clusterHealth,
+  efficiencyValue,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const BROWSER_TIMEOUT = 30000;
+
+describe('Test Object Service Dashboard Page', () => {
+  it('Create an OBC and test equality', async () => {
```
[the remark this answers is not in the document]
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: ack

**Comment by gnehapk** [comment]
It seems that you have added tests for all cards here. Its better to have separate `describe` block for each card as looking at this PR, its not easy to identify test for each card. @bipuladh

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (umangachapagain) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Yes, it shouldn't matter. But, the test report will look cleaner and can suggest on corner cases.
I just saw a bug where Health Check failed after creating OBC.

In fact, I think we should do a Health Check in beforeAll and afterAll to make sure we started and ended healthy. What do you think?

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts (hunk 1 above)
Comment (umangachapagain) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: Please add a comment to the code.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+describe('Tests Object Service Dashboard Page', () => {
+  beforeAll(async () => {
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: We need to start from this page for each of the `it` blocks. So, wouldn't this be `beforeEach` ?
I think `beforeAll` just does a one time setup at the beginning of the tests.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+    await browser.get(`${appHost}/dashboards/object-service`);
+  });
+  it('Creates an Object Bucket Claim and test equality', async () => {
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]:
```suggestion
  it('Creates an Object Bucket Claim and tests equality', async () => {
```

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
+    await browser.sleep(10000);
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -n ${testName} -f -`);
+  });
+
+  it('Checks if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.get(`${appHost}/dashboards/object-service`);
```
Comment (umangachapagain) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Using `beforeEach` above will allow you to remove this.

**Comment by umangachapagain** [comment]
Run linter to ensure uniform formatting.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts (hunk 5 above)
[the remark this answers is not in the document]
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.ts]: I guess the objective of the test is to catch an unhealthy state at some point. I don't think if an unhealthy MCG creates an OBC that is an issue. If unhealthy MCGs should not be able to create OBCs. We should file a separate bug for that. 
Should we really worry about Health being degraded at a certain point? Again the question that arises is what are we testing? If it's whether health can be affected by other components or is it that all the components are actually working as they are supposed to? Because if we are testing the health parameter extensively then we need to take in all the variables that affect Health. 
If Health is degraded when creating an OBC(or any other resource) then should that be concerning from a UI perspective? Technically the UI is still intact and it's reporting a state of the resource correctly.  The reason why it's health got degraded IMO is out of the scope.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 7 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: If we are not changing pages do we need beforeEach?

**Comment by bipuladh** [comment]
@umangachapagain this code has been linted. Hence the frontend tests are passing.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 7 above)
Comment (umangachapagain) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: It's better to ensure that you are on the correct page before you start.
Also, your code already does a `get` on every `it` block, which is why I suggested a `beforeEach`.
If you think, `beforeAll` is enough, remove `get` calls from all the other `it` blocks.

**Comment by umangachapagain** [comment]
> @umangachapagain this code has been linted. Hence the frontend tests are passing.

But there are some inconsistent new line issues between line 25 and 26, and 28 and 29.

**Comment by bipuladh** [comment]
@umangachapagain agreed. IMO these changes will make more sense when they are accompanied by the use case. And the owners of those particular components might be able to give a better approach(possibly using a different tag). These changes aren't structural changes but are only the addition of identifier tags which would not change the layout/behaviour of these components and would not make sense to add without any pretext to add them.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Creates an Object Bucket Claim and tests equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
```
Comment (rhrazdil) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: `execSync` is synchronous, no need to `await`.
Also, no need to specify `-n ${testname}`, namespace is already defined in metadata of the object.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 11]
```diff
@@ -0,0 +1,107 @@
...
+    await browser.sleep(10000);
```
Comment (rhrazdil) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: It would be must nicer to wait for some condition. You can take a look here how simple waiting method can look: https://github.com/openshift/console/blob/41bb8cb8354ab91d18377535c17e070c7e5a0c75/frontend/packages/console-shared/src/test-utils/utils.ts#L135
and here how it's used:
https://github.com/openshift/console/blob/41bb8cb8354ab91d18377535c17e070c7e5a0c75/frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts#L130

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 12]
```diff
@@ -0,0 +1,107 @@
...
+    text = await obcCount.getText();
+    const finalCount = text.substring(0, 1);
+    expect(Number(finalCount)).toEqual(Number(initialCount) + 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl delete -n ${testName} -f -`);
```
Comment (rhrazdil) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: unnecessary await and namespace.

File: frontend/packages/noobaa-storage-plugin/integration-tests/views/noobaaDashboardPage.view.ts [hunk 13]
```diff
@@ -0,0 +1,12 @@
+import { $, element, by } from 'protractor';
+
+export const obcCount = element(by.css('div[aria-label="Object Bucket Claim count"]'));
```
Comment (rhrazdil) [inline frontend/packages/noobaa-storage-plugin/integration-tests/views/noobaaDashboardPage.view.ts]: `$()` is an alias for `element(by.css())`, so these can be simplified by just using `$()`.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 14]
```diff
@@ -0,0 +1,107 @@
...
+  });
+});
+
+describe('Tests Status Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks if Multi Cloud Gateway is in a healthy state', async () => {
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: I don't find this valid. The integration test should say the state of multi-cloud gateway is reflected properly. We should get the state from noobaa command and check this. Noobaa can come up in an unhealthy state and in that case, this test will fail.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 15]
```diff
@@ -0,0 +1,107 @@
...
+  it('Checks if Efficiency Ratio is in acceptable data range', async () => {
+    await browser.wait(until.visibilityOf(efficiencyValue));
+    const effValue = await efficiencyValue.getText();
+    const [ratioA, ratioB] = effValue.split(':');
+    const [numA, numB] = [Number(ratioA), Number(ratioB)];
+    expect(numA).toBeGreaterThan(0);
+    expect(numB).toBeGreaterThan(0);
+  });
+
+  it('Checks for savings value to be in acceptable data range', async () => {
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Again not a valid test. As the saving can be 0 for the initial thing and it's not necessary that savings always happens and depends on data that's present.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 16]
```diff
@@ -0,0 +1,107 @@
...
+    }
+  });
+});
+
+describe('Tests Resource Providers card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Checks for resource provider to have at least 1 AWS resource', async () => {
```
Comment (cloudbehl) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: What if noobaa deployed on vmware?

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: kubevirt tests just can't be run in our local deployments.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 15 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: So we accept no savings too.

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: The point is savings should have some value we don't accept unavailable.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 14 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: we cannot assume that the noobaa command line is available.

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: What possible things are there that can cause Noobaa to go unhealthy? If it can go unhealthy due to some tests then we should run this before that. But in an ideal deployment of Noobaa, I am assuming that Noobaa would come up healthy. Is this a safe assumption? If not why cannot Noobaa come up healthy?

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 17]
```diff
@@ -0,0 +1,108 @@
...
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import { waitFor } from '@console/ceph-storage-plugin/integration-tests/utils/helpers';
+import {
+  obcCount,
+  efficiencyValue,
+  savingsValue,
+  healthOfMCG,
+  resourceProviders,
+  noobaaCount,
```
Comment (gnehapk) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Alpha imports.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 18]
```diff
@@ -0,0 +1,108 @@
...
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Tests at least one noobaa bucket is present', async () => {
```
Comment (gnehapk) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Minor : It should be `Test` instead of `Tests`

File: frontend/packages/console-shared/src/test-utils/utils.ts [hunk 19]
```diff
@@ -157,3 +157,10 @@ export function searchYAML(needle: string, name: string, namespace: string, kind
   const result = execSync(`kubectl get -o yaml -n ${namespace} ${kind} ${name}`).toString();
   return result.search(needle) >= 0;
 }
+
+export const waitFor = (elem: any, target: string) => {
```
Comment (afreen23) [inline frontend/packages/console-shared/src/test-utils/utils.ts]: Use proper type
```suggestion
export const waitFor = (element; any, target: string) => {
```

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 20]
```diff
@@ -0,0 +1,108 @@
...
+import {
+  efficiencyValue,
+  healthOfMCG,
+  noobaaCount,
+  obcCount,
+  resourceProviders,
+  savingsValue,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: This can be a part of mock.ts file

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 21]
```diff
@@ -0,0 +1,108 @@
...
+    }
+  });
+});
+
+describe('Test Resource Providers card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if resource provider has at least 1 AWS resource', async () => {
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: As said it can be on other infra, you should check if it has atleast one resource, not for any specific resource.
```suggestion
  it('Check if resource provider has at least 1 resource', async () => {
```

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 22]
```diff
@@ -0,0 +1,108 @@
...
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Create an Object Bucket Claim and test equality', async () => {
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Why this test case is needed. We are having this tested in OB/OBC test cases

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 20 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: I can refactor that later.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 21 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: In VMWare would there be a resource by default? If so then which cloud provider?

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Tests should not care about that, but rather on the count. There will always be one default backing store wherever the noobaa is deployed with ocs. On vmware it will be ceph.

Comment (afreen23) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts]: Are you creating these test cases by yourself ?

**Comment by afreen23** [comment]
A reason for closing and reopening would help here @bipuladh .

File: frontend/packages/console-shared/src/test-utils/utils.ts (hunk 19 above)
Comment (bipuladh) [reply frontend/packages/console-shared/src/test-utils/utils.ts:197]: There is no type definition for Element Selectors.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 23]
```diff
@@ -0,0 +1,106 @@
...
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Test at least one noobaa bucket is present', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
```
Comment (gnehapk) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts:28]: Since this is already checked in beforeAll, why do you need to check it again here?

Comment (gnehapk) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts:28]: Its present at other places too

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts [hunk 24]
```diff
@@ -0,0 +1,106 @@
+import { execSync } from 'child_process';
```
Comment (gnehapk) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts:1]: The file should be named to something like `object-service-dashboard` instead of `overviewDashboard`

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts (hunk 23 above)
Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts:28]: yes. For some reason, it fails without this.

File: frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectSerivceDashboard.scenario.ts [hunk 25]
```diff
@@ -0,0 +1,106 @@
...
+describe('Test Status Card', () => {
+  beforeAll(async () => {
+    await loadAndWait();
+  });
+
+  it('Check if Multi Cloud Gateway is in a healthy state', async () => {
+    await browser.wait(until.and(untilNoLoadersPresent));
+    await browser.wait(until.visibilityOf(healthOfMCG));
+    const color = healthOfMCG.getAttribute('fill');
+    expect(color).toBe('#486b00');
```
Comment (ebondare) [inline frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectSerivceDashboard.scenario.ts:57]: Wouldn't it be nicer to import the code for the green color from https://github.com/openshift/console/blob/master/frontend/packages/ceph-storage-plugin/integration-tests/utils/consts.ts ?

Comment (bipuladh) [reply frontend/packages/noobaa-storage-plugin/integration-tests/tests/objectSerivceDashboard.scenario.ts:57]: Makes sense. Thanks.
